addons/vscode/package.json

```diff
@@ -139,6 +139,14 @@
           "default": "",
           "description": "Path to typst-preview executable. If not set, the extension will use bundled typst-preview. If bundled binary is not found, it will use typst-preview installed in PATH"
         },
+        "typst-preview.typstInputs": {
```

Comment:
maybe `typst-preview.sys-inputs`?

Reply:
Not sure about the naming convention here. Maybe `SysInputs`..? cc @Myriad-Dreamin bikeshedding time

Reply:
All settings should be `camelCase` for consistency.

Reply:
`sysInputs` seems great, I can go ahead with this.